pages/linux/netcat.md

```diff
@@ -0,0 +1,36 @@
+# netcat
+
+> The `nc` (or `netcat`) utility is used for just about anything under the sun involving TCP, UDP, or UNIX-domain sockets. It can open TCP connections, send UDP packets, listen on arbitrary TCP and UDP ports, do port scanning.
+> More information: <http://man.openbsd.org/OpenBSD-current/man1/nc.1>.
+
+- Normal Syntax:
+
+`nc {{options}} {{host}} {{port}}`
+
+- Use IPv4 addressing only:
+
+`nc -4 {{host}} {{port}}`
```

Comment:
```suggestion
- Use IPv[4] addressing only:

`nc -4 "{{host}}" "{{port}}"`
```

Optional fix: mnemonic can be used for `-4`.
Optional fix: placeholders can be quoted to protect from wrong POSIX-compatible shell interpretation.